bin/pca-import

```diff
@@ -170,27 +172,31 @@ def import_assessment(request, source, force=False):
     return True
 
 
-def import_json_file(filename, force, customerImport, assessmentImport):
+def import_json_file(filename, force, customer_import, assessment_import, user_report_import):
     f = open(filename, "rb")
     request = json.load(f)
     f.close()
-    if customerImport:
+    if customer_import:
         return import_customer(request, filename, force)
-    elif assessmentImport:
+    elif assessment_import:
         return import_assessment(request, filename, force)
+    elif user_report_import:
+        return import_user_report(request, filename, force)
 
 
-def import_json_stdin(force, customerImport, assessmentImport):
+def import_json_stdin(force, customer_import, assessment_import, user_report_import):
```

Comment:
The comment above about `import_json_file()` also applies here.

Reply:
See a8014ba675ba14a104152964cb72cf7ce0bb7127